db/data_migration/20170217094518_republish_problematic_translations.rb

```diff
@@ -0,0 +1,662 @@
+  /government/world-location-news/179503.zh-tw
+  /government/world-location-news/186212.zh-tw
+  /government/world-location-news/188502.zh-tw
+  /government/world-location-news/189734.zh-tw
+  /government/world-location-news/189739.zh-tw
+  /government/world-location-news/192841.zh-tw
+  /government/world-location-news/197379.zh-tw
+  /government/world-location-news/197856.zh-tw
+  /government/world-location-news/197857.zh-tw
+  /government/world-location-news/203276.zh-tw
+  /government/world-location-news/209141.zh-tw
+  /government/world-location-news/210389.zh-tw
+  /government/world-location-news/210510.zh-tw
+  /government/world-location-news/210520.zh-tw
+  /government/world-location-news/210686.zh-tw
+  /government/world-location-news/213347.zh-tw
+  /government/world-location-news/215732.zh-tw
+  /government/world-location-news/216080.zh-tw
+  /government/world-location-news/220114.zh-tw
+  /government/world-location-news/220750.zh-tw
+  /government/world-location-news/221235.zh-tw
+  /government/world-location-news/221645.zh-tw
+  /government/world-location-news/222109.zh-tw
+  /government/world-location-news/228361.zh-tw
+  /government/world-location-news/229395.zh-tw
+  /government/world-location-news/231480.zh-tw
+  /government/world-location-news/232909.zh-tw
+  /government/world-location-news/238023.zh-tw
+  /government/world-location-news/240063.zh-tw
+  /government/world-location-news/245635.zh-tw
+  /government/world-location-news/249089.zh-tw
+  /government/world-location-news/250984.zh-tw
+  /government/world-location-news/251447.zh-tw
+  /government/world-location-news/252606.zh-tw
+  /government/world-location-news/252778.zh-tw
+  /government/world-location-news/254154.zh-tw
+  /government/world-location-news/256970.zh-tw
+  /government/world-location-news/260960.zh-tw
+  /government/world-location-news/262114.zh-tw
+  /government/world-location-news/263388.zh-tw
+  /government/world-location-news/264597.zh-tw
+  /government/world-location-news/268479.zh-tw
+  /government/world-location-news/268483.zh-tw
+  /government/world-location-news/276559.zh-tw
+  /government/world-location-news/277623.zh-tw
+  /government/world-location-news/282153.zh-tw
+  /government/world-location-news/283286.zh-tw
+  /government/world-location-news/289225.zh-tw
+  /government/world-location-news/290168.zh-tw
+  /government/world-location-news/292788.zh-tw
+  /government/world-location-news/294626.zh-tw
+  /government/world-location-news/296968.zh-tw
+  /government/world-location-news/247263.zh
+  /government/world-location-news/272824.zh
+  /government/world-location-news/274722.zh
+  /government/world-location-news/282389.zh
+  /government/world-location-news/282404.zh
+  /government/world-location-news/286926.zh
+  /government/world-location-news/291036.zh
+)
+
+base_paths = base_paths_200_302 + base_paths_bad_wlna
+
+Edition.where(base_path: base_paths).joins(:document).distinct.pluck(:id).each do |id|
```

Comment:
This doesn't work.  Firstly, `Edition` doesn't have a `base_path` field in Whitehall either so you can't query on it.

Secondly, if the query did work we need `document_id` to republish with and this is returning `Edition#id`. I also don't think the join/distinct bit is necessary as it doesn't help with the `document_id`. Once you get the right query to return an edition AR relation you can just `pluck(:document_id).uniq`.